kubernetes/src/main/resources/examples/kubernetes.yaml

```diff
@@ -23,9 +23,13 @@ services:
     name: "kubernetes"
     brooklyn.config:
       start.timeout: 30m
+      etcd.initial.size: 3
+      docker.recovery.stabilizationDelay: 10s
+      docker.recovery.failOnRecurringFailuresInThisDuration: 5m
+      kubernetes.debug: true
+      kubernetes.version: 1.4.0
       kubernetes.initial.size: 4
       kubernetes.max.size: 16
-      etcd.initial.size: 3
```

Comment:
Why remove `etcd.initial.size: 3`? That is not done in https://github.com/brooklyncentral/clocker/pull/329 or https://github.com/brooklyncentral/clocker/pull/330